doc/classes/ColorRect.xml

```diff
@@ -1,25 +1,17 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="ColorRect" inherits="Control" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Colored rectangle.
+		A control that displays a solid color rectangle.
 	</brief_description>
 	<description>
-		Displays a rectangle filled with a solid [member color]. If you need to display the border alone, consider using [ReferenceRect] instead.
+		Displays a rectangle filled with a solid [member color]. If you need to display the border alone, consider using a [PanelContainer] instead.
```

Comment:
I think suggesting ReferenceRect was correct. It can be displayed at runtime too.
Also PanelContainer is a container. It's a completely different thing. If anything, it should be Panel (can be in addition to ReferenceRect).

Reply:
Idk, I don't feel good telling people to use something that's intended as an editor hint. The ability to show up in-game wasn't even present in older versions, and editor-only is by default. Even the "in-game display" I would say is also meant for debugging purposes, not to have borders in the final product.

I'll leave this up as unresolved for now but I'll fix the PanelContainer mistake.